bin/lib/recreate_database.py

```diff
@@ -14,11 +16,11 @@ def main():
     print '****************************************************'
     print
     while True:
-        response = raw_input('Proceed [y/n]? ')
-        if response.lower() == 'y':
+        response = raw_input('Proceed [yes/no]? ')
```

Comment:
This is nitpicking but I think you should leave this as `y/n`, or at least accept `y` as an alias for `yes`. My reasoning for this is that the `yes` Unix command outputs `y` by default. One of my own scripts has a line that reads `yes | bash bin/recreate_database.sh`, haha. Plus, a lot of users are conditioned by other applications to just type `y`.

If you're really concerned that someone will drop the database unintentionally then I'll concede but I thought I'd make my case anyway.